cmd/ocagent/Dockerfile

```diff
@@ -0,0 +1,9 @@
+# Use a helper to create an emtpy configuration since the agent requires such file
+FROM alpine:3.7 as helper
```

Comment:
this is a bit strange
maybe we can get config from environment vars instead of mandating config file for startup (and fall back to defaults).

Reply:
Yeah, right now the agent requires the file to be present (even an empty one). We should use something like Viper combined with Cobra to allow flexibility in this regard: no file is required by default but settings can be specified in a file or cli as convenient. Tracking #158